docs/mkdocs.yml

```diff
@@ -17,6 +17,7 @@ nav:
           - Custom strategy graphs: custom-strategy-graphs.md
           - Pre-defined nodes and components: nodes-and-components.md
           - Parallel node execution: parallel-node-execution.md
+          - Data transfer between nodes: data-transfer-between-nodes.md
```

Comment:
I'd suggest to name it as agent data storage or something, as in koog data transferring between nodes in mainly about nodes input-output